pkg/destroy/aws/aws.go

```diff
@@ -996,6 +998,30 @@ func deleteEC2NATGatewaysByVPC(ctx context.Context, client *ec2.EC2, vpc string,
 	return err
 }
 
+func deleteEC2PlacementGroup(ctx context.Context, client *ec2.EC2, id string, logger logrus.FieldLogger) error {
+	response, err := client.DescribePlacementGroupsWithContext(ctx, &ec2.DescribePlacementGroupsInput{
+		GroupIds: []*string{aws.String(id)},
+	})
+	if err != nil {
+		if err != nil {
+			if err.(awserr.Error).Code() == "InvalidPlacementGroup.Unknown" {
```

Comment:
This magic string came from a manual test, I tried to delete a placement group which I had already deleted and got this response body, so I think I have the correct string here based on this example
```
<?xml version="1.0" encoding="UTF-8"?>
<Response><Errors><Error><Code>InvalidPlacementGroup.Unknown</Code><Message>The Placement Group 'pg-04e435e5cedc43563' is unknown.</Message></Error></Errors><RequestID>cf6e03f4-35fe-40cc-bd20-15bd76759d40</RequestID></Response>
```